### ISSUE TITLE
icon for medkit tool doesn't exist, it just duplicates the last number of the ammo count

### ISSUE THREAD

**Issue by Glitch31415** [wootguy/SevenKewp#39]
![Screenshot from 2024-11-30 01-46-58](https://github.com/user-attachments/assets/ebac1aea-467d-4cf2-883c-d04585df493d)
![Screenshot from 2024-11-30 01-46-54](https://github.com/user-attachments/assets/66352255-82fd-4f9f-9178-b07256f99155)

**Comment by wootguy** [comment]
which resolution is the game running at?

**Comment by Glitch31415** [comment]
1440p
also, the hud item for selecting the medkit is just the wrench and the grappler overlapped
![Screenshot from 2024-11-30 13-43-24](https://github.com/user-attachments/assets/13020be3-cc93-45e9-b37f-0d9ccdac20f2)

**Comment by Glitch31415** [comment]
i think i might need to update my sevenkewp data folder again

**Comment by Glitch31415** [comment]
didn't help

**Comment by wootguy** [comment]
I use the same resolution so I think you need to update these files:
```
sprites/hlcoop/weapon_medkit.spr
sprites/hlcoop/weapon_medkit.txt
hlcoop.gmr
```
or precache weapon_medkit if you're using impulse 101 to test.

**Comment by Glitch31415** [comment]
so i found the issue, for some fucking reason the rehlds and sevenkewp submodules aren't staying up to date on my fork!! github is so stupid sometimes

**Comment by wootguy** [comment]
yeah I think you need to run a separate command to update the submodules

**Comment by Glitch31415** [comment]
yep, this is my full command to generate the files for my server:
```
cd .. ; rm -rf sevenkewp-sandbox ; git clone --recurse-submodules https://github.com/Glitch31415/sevenkewp-sandbox.git ; cd sevenkewp-sandbox ; git submodule foreach git pull origin master ; git add . ; git commit -m "updated submodules" ; git push origin master ; chmod +x ./scripts/build_game_and_plugins.sh ; ./scripts/build_game_and_plugins.sh
```

**Comment by Glitch31415** [comment]
well, everything is updated, but the issues are still there
